packages/contracts-bedrock/deploy/017-SystemConfigImpl.ts

```diff
@@ -18,6 +22,7 @@ const deployFn: DeployFunction = async (hre) => {
       batcherHash,
       hre.deployConfig.l2GenesisBlockGasLimit,
       hre.deployConfig.p2pSequencerAddress,
+      Artifact__OptimismPortalProxy.address,
```

Comment:
Maybe it's worth adding a check to the l2-check script, to verify it's configured correctly.